app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -39,7 +39,10 @@ def authenticate_client
       bearer, csp_id, token = request.authorization&.split(' ', 3)
       if bearer != 'Bearer' || !valid_auth_tokens.include?(token) ||
          csp_id != IdentityConfig.store.irs_attempt_api_csp_id
+        analytics.irs_attempts_api_authentication(success: false)
         render json: { status: 401, description: 'Unauthorized' }, status: :unauthorized
+      else
+        analytics.irs_attempts_api_authentication(success: true)
```

Comment:
Wondering if there might be some possibility to avoid doubling up on `irs_attempts_api_authentication` and `irs_attempts_api_events` for each call. I don't suppose it could make sense to handle this as part of the existing `irs_attempts_api_events` ?

Reply:
Yeah, I wasn't sure about that. You think we could just log `irs_attempts_api_events(success: false)` on the authentication failure piece?

Reply:
That was my first thought, yeah. But I see we do already handle some unsuccessful events with that, so unclear if we'd want to disambiguate. If we did, maybe we want to introduce form + validation + `errors` hash?

Reply:
We'd want to disambiguate yeah. A form feels a little heavy for now, so I've added an `authenticated` parameter to the existing analytics event. I'm definitely open to the form validation, but am slightly unsure about the complexity of being able to build a metric around the errors hash vs. a boolean in the event.